pom.xml

```diff
@@ -21,16 +21,15 @@
         <maven.compiler.target>11</maven.compiler.target>
         <java.version>11</java.version>
 
-        <spring-version>2.6.1</spring-version>
+        <spring-version>2.6.2</spring-version>
         <spring-retry-version>1.3.1</spring-retry-version>
         <ojdbc8-version>21.4.0.0</ojdbc8-version>
         <lombok-version>1.18.22</lombok-version>
-        <springdoc-version>1.6.2</springdoc-version>
+        <springdoc-version>1.6.3</springdoc-version>
         <springfox-version>3.0.0</springfox-version>
         <modelmapper-version>2.4.5</modelmapper-version>
         <log4j2.version>2.16.0</log4j2.version>
         <jackson-datatype-version>2.13.1</jackson-datatype-version>
-        <h2-version>1.4.200</h2-version>
```

Comment:
why remove `h2` version ?

Reply:
we have a [security alert ](https://github.com/pagopa/pagopa-api-config/security/dependabot)